pytac/element.py

```diff
@@ -172,7 +177,7 @@ def get_pv_name(self, field, handle='*'):
         try:
             return self._devices[field].get_pv_name(handle)
         except KeyError:
-            raise PvException('Element has no device for field {}'.format(field))
```

Comment:
Should this exception be renamed to something more generic, CsException perhaps?